Meter.h

```diff
@@ -111,7 +111,7 @@ struct Meter_ {
    unsigned int param;
    GraphData drawData;
    int h;
-   int columnWidthCount;      /**< only used internally by the Header */
+   size_t columnWidthCount;      /**< only used internally by the Header */
```

Comment:
Can't understand the reason why expanding the bit width of this `columnWidthCount` to 64. Would it save code size by saving some cast instructions?